packages/core/src/Table/stories/TableSection/CompleteTableSection.tsx

```diff
@@ -32,8 +32,9 @@ const EmptyRow = () => (
 );
 
 export const CompleteTableSection = () => {
+  const totalRows = 64;
   const columns = useMemo(() => getColumns(), []);
-  const [data, setData] = useState(makeData(64));
+  const [data, setData] = useState(makeData(totalRows));
```

Comment:
We should us `data.length`, not `totalRows`. Rows could be added or removed
```suggestion
  const [data, setData] = useState(makeData(64));
```